bottlerocket/agents/src/bin/eks-resource-agent/eks_provider.rs

```diff
@@ -144,15 +247,6 @@ impl ClusterConfig {
                     )?)
                     .context(Resources::Clear, "Unable to serialize eksctl config.")?;
 
```

Comment:
It seems like you just broke the ability to pass in a file for `ClusterConfig`? Can you explain why you removed this?

Reply:
I didn't change this part of the code, used to add some code here later on I recovery to it original state.